samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/doc/FakeApi.md

```diff
@@ -65,53 +64,6 @@ No authorization required
 
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
-# **fakeHttpSignatureTest**
-> fakeHttpSignatureTest(pet, query1, header1)
-
-test http signature authentication
-
-### Example
-```dart
-import 'package:openapi/api.dart';
-// TODO Configure HTTP basic authorization: http_signature_test
-//defaultApiClient.getAuthentication<HttpBasicAuth>('http_signature_test').username = 'YOUR_USERNAME'
-//defaultApiClient.getAuthentication<HttpBasicAuth>('http_signature_test').password = 'YOUR_PASSWORD';
```

Comment:
note for discussion: when http_signature_test is kept, the samples contain this incorrect code example